__tests__/basic.js

```diff
@@ -136,6 +136,7 @@ test('thumbnail card with a long title and richCardWrapTitle set to default valu
 test('absolute timestamp', async () => {
   const activities = [
     {
+      channelData: {},
```

Comment:
As this data structure is dumped into Redux store in Web Chat, it bypassed our patch code.

Our patch code normalizes the activity from DLJS, say:
- It must have `attachments` array, but the array can be empty
- It must have `channelData`, but it can be empty

So, if we dump it into Redux store, the data needs to be clean. In this case, `channelData` need to be `{}`.